# HG changeset patch
# User Rik <rik@octave.org>
# Date 1379624591 25200
#      Thu Sep 19 14:03:11 2013 -0700
# Node ID 9e4e24a0a118ea61dc20122970603287317024ee
# Parent  2fee3022ac2f08a7ee332d2c8c0f69f53195fb1d
doc: Use common language for xlim, ylim, zlim docstrings.

* scripts/plot/xlim.m, scripts/plot/ylim.m, scripts/plot/zlim.m:
Use common language for xlim, ylim, zlim docstrings.

diff --git a/scripts/plot/xlim.m b/scripts/plot/xlim.m
--- a/scripts/plot/xlim.m
+++ b/scripts/plot/xlim.m
@@ -18,28 +18,27 @@
 
 ## -*- texinfo -*-
 ## @deftypefn  {Function File} {@var{xlimits} =} xlim ()
 ## @deftypefnx {Function File} {@var{xmode} =} xlim ("mode")
 ## @deftypefnx {Function File} {} xlim ([@var{x_lo} @var{x_hi}])
 ## @deftypefnx {Function File} {} xlim ("auto")
 ## @deftypefnx {Function File} {} xlim ("manual")
 ## @deftypefnx {Function File} {} xlim (@var{hax}, @dots{})
-## Query or set the limits of the x-axis of the current plot.
+## Query or set the limits of the x-axis for the current plot.
 ##
 ## Called without arguments @code{xlim} returns the x-axis limits of the
 ## current plot.  With the input query @qcode{"mode"}, return the current
-## x-limit calculation mode which is either @qcode{"auto"} or
-## @qcode{"manual"}.
+## x-limit calculation mode which is either @qcode{"auto"} or @qcode{"manual"}.
 ##
 ## If passed a 2-element vector [@var{x_lo} @var{x_hi}], the limits of the
-## x-axis are set to these values.
+## x-axis are set to these values and the mode is set to @qcode{"manual"}.
 ##
-## The current plotting mode can be set by passing either @qcode{"auto"} or
-## @qcode{"manual"} as the argument.
+## The current plotting mode can be changed by using either @qcode{"auto"}
+## or @qcode{"manual"} as the argument.
 ##
 ## If the first argument @var{hax} is an axes handle, then operate on
 ## this axis rather than the current axes returned by @code{gca}.
 ## @seealso{ylim, zlim, axis, set, get, gca}
 ## @end deftypefn
 
 function retval = xlim (varargin)
   ret = __axes_limits__ ("xlim", varargin{:});
diff --git a/scripts/plot/ylim.m b/scripts/plot/ylim.m
--- a/scripts/plot/ylim.m
+++ b/scripts/plot/ylim.m
@@ -18,28 +18,27 @@
 
 ## -*- texinfo -*-
 ## @deftypefn  {Function File} {@var{ylimits} =} ylim ()
 ## @deftypefnx {Function File} {@var{xmode} =} ylim ("mode")
 ## @deftypefnx {Function File} {} ylim ([@var{y_lo} @var{y_hi}])
 ## @deftypefnx {Function File} {} ylim ("auto")
 ## @deftypefnx {Function File} {} ylim ("manual")
 ## @deftypefnx {Function File} {} ylim (@var{hax}, @dots{})
-## Query or set the limits of the y-axis of the current plot.
+## Query or set the limits of the y-axis for the current plot.
 ##
 ## Called without arguments @code{ylim} returns the y-axis limits of the
 ## current plot.  With the input query @qcode{"mode"}, return the current
-## y-limit calculation mode which is either @qcode{"auto"} or
-## @qcode{"manual"}.
+## y-limit calculation mode which is either @qcode{"auto"} or @qcode{"manual"}.
 ##
 ## If passed a 2-element vector [@var{y_lo} @var{y_hi}], the limits of the
-## y-axis are set to these values.
+## y-axis are set to these values and the mode is set to @qcode{"manual"}.
 ##
-## The current plotting mode can be set by passing either @qcode{"auto"} or
-## @qcode{"manual"} as the argument.
+## The current plotting mode can be changed by using either @qcode{"auto"}
+## or @qcode{"manual"} as the argument.
 ##
 ## If the first argument @var{hax} is an axes handle, then operate on
 ## this axis rather than the current axes returned by @code{gca}.
 ## @seealso{xlim, zlim, axis, set, get, gca}
 ## @end deftypefn
 
 function retval = ylim (varargin)
   ret = __axes_limits__ ("ylim", varargin{:});
diff --git a/scripts/plot/zlim.m b/scripts/plot/zlim.m
--- a/scripts/plot/zlim.m
+++ b/scripts/plot/zlim.m
@@ -18,28 +18,27 @@
 
 ## -*- texinfo -*-
 ## @deftypefn  {Function File} {@var{zlimits} =} zlim ()
 ## @deftypefnx {Function File} {@var{xmode} =} zlim ("mode")
 ## @deftypefnx {Function File} {} zlim ([@var{z_lo} @var{z_hi}])
 ## @deftypefnx {Function File} {} zlim ("auto")
 ## @deftypefnx {Function File} {} zlim ("manual")
 ## @deftypefnx {Function File} {} zlim (@var{hax}, @dots{})
-## Query or set the limits of the z-axis of the current plot.
+## Query or set the limits of the z-axis for the current plot.
 ##
 ## Called without arguments @code{zlim} returns the z-axis limits of the
 ## current plot.  With the input query @qcode{"mode"}, return the current
-## z-limit calculation mode which is either @qcode{"auto"} or
-## @qcode{"manual"}.
+## z-limit calculation mode which is either @qcode{"auto"} or @qcode{"manual"}.
 ##
 ## If passed a 2-element vector [@var{z_lo} @var{z_hi}], the limits of the
-## z-axis are set to these values.
+## x-axis are set to these values and the mode is set to @qcode{"manual"}.
 ##
-## The current plotting mode can be set by passing either @qcode{"auto"} or
-## @qcode{"manual"} as the argument.
+## The current plotting mode can be changed by using either @qcode{"auto"}
+## or @qcode{"manual"} as the argument.
 ##
 ## If the first argument @var{hax} is an axes handle, then operate on
 ## this axis rather than the current axes returned by @code{gca}.
 ## @seealso{xlim, ylim, axis, set, get, gca}
 ## @end deftypefn
 
 function retval = zlim (varargin)
   ret = __axes_limits__ ("zlim", varargin{:});
